kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/aggregation/settings/dsl/KoverSettingsExtension.kt

```diff
@@ -17,11 +23,268 @@ public interface KoverSettingsExtension {
 }
 
 @KoverGradlePluginDsl
-public interface ReportsSettings {
+public interface ReportsSettings: ReportFiltersSettings {
+    val verify: VerifySettings
+
+    fun verify(action: Action<VerifySettings>) {
+        action.execute(verify)
+    }
+}
+
+@KoverGradlePluginDsl
+public interface ReportFiltersSettings {
+    fun clearFilters() {
+        includedProjects.empty()
+        excludedProjects.empty()
+        excludedClasses.empty()
+        includedClasses.empty()
+        excludesAnnotatedBy.empty()
+        includesAnnotatedBy.empty()
+        includesInheritedFrom.empty()
+        excludesInheritedFrom.empty()
+    }
+
     val includedProjects: SetProperty<String>
     val excludedProjects: SetProperty<String>
     val excludedClasses: SetProperty<String>
     val includedClasses: SetProperty<String>
     val excludesAnnotatedBy: SetProperty<String>
     val includesAnnotatedBy: SetProperty<String>
+    val includesInheritedFrom: SetProperty<String>
+    val excludesInheritedFrom: SetProperty<String>
+}
+
+
+@KoverGradlePluginDsl
+public interface VerifySettings {
+    public val rules: ListProperty<VerificationRuleSettings>
+
+    /**
+     * Add new coverage verification rule to check.
+     */
+    public fun rule(action: Action<VerificationRuleSettings>)
+
+    /**
+     * Add new named coverage verification rule to check.
+     *
+     * The name will be displayed in case of a verification error.
+     */
+    public fun rule(name: String, action: Action<VerificationRuleSettings>)
+
+    /**
+     * In case of a verification error, print a message to the log with the warn level instead of the Gradle task execution error.
+     *
+     * Gradle task error if `false`, warn message if `true`.
+     *
+     * `false` by default.
+     */
+    public val warningInsteadOfFailure: Property<Boolean>
+}
+
+@KoverGradlePluginDsl
+public interface VerificationRuleSettings {
+    /**
+     * Name of the rule. Will be displayed in case of a verification error.
+     *
+     * Empty by default.
+     */
+    public val name: Property<String>
+
+    /**
+     * Specifies that the rule is checked during verification.
+     *
+     * `false` by default.
+     */
+    public val disabled: Property<Boolean>
+
+    /**
+     * Specifies by which entity the code for separate coverage evaluation will be grouped.
+     * [GroupingEntityType.APPLICATION] by default.
+     */
+    public val groupBy: Property<GroupingEntityType>
+
+    public val filters: ReportFiltersSettings
+
+    public fun filters(action: Action<ReportFiltersSettings>) {
+        action.execute(filters)
+    }
+
+    public val bounds: ListProperty<BoundSettings>
```

Comment:
Also undocumented (and `fun filters(action: Action<ReportFiltersSettings>)` too)